tawazi/_dag/digraph.py

```diff
@@ -6,7 +6,7 @@
 from loguru import logger
 from networkx import NetworkXNoCycle, find_cycle
 
-from .consts import Identifier
+from tawazi.consts import Identifier
```

Comment:
I'd rather have a relative import here to limit the potential appearance of the error 'attempting import from currently unitiltialized module'

Reply:
Ruff is forcing this pattern: https://beta.ruff.rs/docs/rules/relative-imports/
plus if we have this error it usually indicates a circular import...

Reply:
ruff actually looks like it accepts both in the doc you sent me no ?

Reply:
the doc says: Absolute imports, or relative imports from siblings, are recommended by [PEP 8](https://peps.python.org/pep-0008/#imports)

Here consts is not a a sibling but rather an uncle

Reply:
this is why  ruff is not happy

Reply:
oh

Reply:
ok I'll resolve this one